### ISSUE TITLE
Package path ./radar is not exported in 5.1.5

### ISSUE THREAD

**Issue by gguynn** [amcharts/amcharts5#300]
Creating an optimized production build...
Failed to compile.

Module not found: Error: Package path ./radar is not exported from package /node_modules/@amcharts/amcharts5

**Comment by martynasma** [comment]
Are you sure? I just created an empty project with amCharts 5.1.5 in it and `radar.js` is there in `node_modules`.

**Comment by gguynn** [comment]
@martynasma

steps I followed:
- removed package-lock and node_modules folder
- re-installed
- ran a build

this happened in both local and CI.

I didn't investigate any further, instead, I just pinned amcharts 5.1.4

seems only the "radar" bit threw the error.  I'll be happy to investigate futher.

```
// amcharts
import {
  addLicense as am5core_addLicense,
  Root as am5core_root,
  color as am5core_color,
  Color as am5core_Color,
  percent as am5core_percent,
  Circle as am5core_Circle,
  Label as am5core_Label,
  RoundedRectangle as am5core_RoundedRectangle,
  Tooltip as am5core_Tooltip
} from "@amcharts/amcharts5"
import {
  RadarChart as am5charts_RadarChart,
  RadarColumnSeries as am5charts_RadarColumnSeries,
  AxisRendererCircular as am5charts_AxisRendererCircular,
  AxisRendererRadial as am5charts_AxisRendererRadial
} from "@amcharts/amcharts5/radar"
import {
  CategoryAxis as am5charts_CategoryAxis,
  ValueAxis as am5charts_ValueAxis
} from "@amcharts/amcharts5/xy"
// license
const amchartsLicense: any = process.env.REACT_APP_AMCHARTS_LICENSE
am5core_addLicense(amchartsLicense)
```

**Comment by nseb** [comment]
I have the same problem , radar and xy

**Comment by martynasma** [comment]
It was caused by a recent fix which we're reverting and release a new version shortly. Will let you know.

**Comment by gguynn** [comment]
@martynasma Issue appears resolved in 5.1.6.  Builds are successful in both local and CI.  Thank you!
